changelogs/CHANGELOG_release.md

```diff
@@ -54,6 +54,8 @@ for (const className of classNames) {
 ### Bug Fixes
 - Fixes build for WeChat WeApp, to reduce package size, see [issue/#1331](https://github.com/parse-community/Parse-SDK-JS/issues/1331)
 
+- Fixes base64 formatting, see [issue/#1467](https://github.com/parse-community/Parse-SDK-JS/pull/1467)
```

Comment:
Changelog entries are not required anymore, please revert any changes to the file.